controller.go

```diff
@@ -94,7 +94,7 @@ func newControllerWithVersion(baseURL, apiVersion, apiKey string) (Controller, e
 		Minor: minor,
 	}
 	controller := &controller{client: client, apiVersion: controllerVersion}
-	controller.capabilities, err = controller.readAPIVersionInfo()
+	_, _, controller.capabilities, err = controller.readAPIVersionInfo()
```

Comment:
If we cache the capabilities here, there's less point in `APIVersionInfo` querying fresh and returning potentially (admittedly extremely unlikely) diverging info.

I'd be inclined to cache the version and subversion too, and just return the members in `APIVersionInfo` without hitting the API again.

Reply:
Normally I'd agree, however the purpose here is to have a light weight api call specifically to check connectivity.  Otherwise it'd be a call to return the api endpoint used by the client, and call Ping inside the provided.